apps/ui/src/views/Space/Proposals.vue

```diff
@@ -72,17 +42,22 @@ watch(
       proposalsStore.reset(toSpace.id, toSpace.network);
       proposalsStore.fetch(toSpace.id, toSpace.network, toState);
     }
-
-    if (toSpace.id !== fromSpace?.id) {
-      getVotingPower();
-    }
   },
   { immediate: true }
 );
 
 watch(
-  () => web3.value.account,
-  () => getVotingPower()
+  [props.space, () => web3.value.account, () => web3.value.authLoading],
+  ([toSpace, toAccount, toAuthLoading], [, fromAccount, , ,]) => {
```

Comment:
```suggestion
  ([toSpace, toAccount, toAuthLoading], [, fromAccount]) => {
```

I think we got rid of those before in review 🤔